components/bl-carousel-component/src/subcomponents.js

```diff
@@ -98,38 +98,26 @@ export function CarouselIndicators(props) {
   );
 }
 
-export function NextButton({ disabled, onNextButton }) {
+export function Button({ disabled, onClick, type, Icon }) {
```

Comment:
We can avoid passing Icon, as we already pass the type.
Just use the condition to choose the icon inside the component.

Also, I do not clearly understand why we don't use disable property on the button but use disabling pointer events through styles.

Reply:
At first, I was using disable, but later I found a bug that @Valodya helped me solve with pointer events